projects/ad9361/src/main.c

```diff
@@ -877,14 +881,14 @@ int main(void)
 	if (status < 0)
 		return status;
 	iio_ad9361_get_dev_descriptor(iio_ad9361_desc, &ad9361_dev_desc);
-	status = iio_register(iio_app_desc, ad9361_dev_desc, "ad9361-phy", ad9361_phy,
-			      NULL, NULL);
-	if (status < 0)
-		return status;
 
-	do {
-		status = iio_step(iio_app_desc);
-	} while (true);
+	struct iio_app_device devices[] = {
+		IIO_APP_DEVICE("axi_adc", iio_axi_adc_desc, adc_dev_desc, &read_buff, NULL),
+		IIO_APP_DEVICE("axi_dac", iio_axi_dac_desc, dac_dev_desc, &write_buff, NULL),
+		//IIO_APP_DEVICE("ad9361", ad9361_phy, ad9361_dev_desc, NULL, NULL)
```

Comment:
Let's remove commented code if we don't need it.

Reply:
the commented line is the one that has been causing some issues and I left it there in case anyone wanted to test it